chore(blog): faq article about hackathon

derberg opened the pull request:
The purpose of this article is to explain in detail what is the whole Hackathon all about. It will eventually be linked from the conference website.

derberg committed:
chore(blog): faq article about hackathon

derberg committed:
Merge branch 'master' into hackathon-faq

derberg committed:
added missing answers

derberg committed:
Merge branch 'master' into hackathon-faq

smoya commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+### What is the scope of the submission? What does it mean I finished my solution on time?
+
+- Your solution must be fully open-sourced and stored in a repository under your GitHub profile,
```
Is Gitlab or Bitbucket not allowed?

derberg replied:
if somebody is not using GitHub then I don't see really how we can long term collaborate together in the initiative. What does it give to the hackathon participant to store code on BitBucket or Gitlab? 

For me, it adds more complexity for our community members that will review submissions?

smoya replied:
I think "Github is a must" deserves a bullet point here as it might become a faq as well.

derberg replied:
Makes sense, added, please have a look and suggest improvements if needed 🙇🏼

derberg commented:
@alequetzalli wanna have a look?

@arjungarg07 & @KhudaDad414 you had some questions, some of them I've put in FAQ. Have a look? especially if there are some answers that are not addressed

smoya commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+- Your solution must be fully open-sourced and stored in a repository under your GitHub profile,
+- It cannot integrate with a commercial solution that has no free tier,
+- You must submit it before the deadline. Evaluation of the solution will not take into account any commits pushed to the repository after the deadline,
```
```suggestion
- You must submit it before the deadline. Evaluation of the solution will not take into account any commits pushed to the repository after the deadline
```

derberg committed:
Update pages/blog/hackathon-faq.md

Co-authored-by: Sergio Moya <1083296+smoya@users.noreply.github.com>

smoya commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+### What is the scope of the submission? What does it mean I finished my solution on time?
+
+- Your solution must be fully open-sourced and stored in a repository under your GitHub profile,
```
Per consistency with other points:
```suggestion
- Your solution must be fully open-sourced and stored in a repository under your GitHub profile
```

derberg committed:
more clarification on GitHub

smoya commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+### What is the scope of the submission? What does it mean I finished my solution on time?
+
+- Your solution must be fully open-sourced and stored in a repository under your GitHub profile. Please do not use Bitbucket or GitLab as it will increase complexity for our community members that will review submissions,
```
I suggest you to **only** use one ending character style when creating bullets but not mixing them across the doc:

- End with period
- End without period
- Whatever

derberg committed:
add period for every bullet point

derberg commented:
@smoya I decided to go with `.` on every bullet point

dalelane commented:
I think it would be good to add a question about the intellectual property / ownership of submitted hacks.

Example text I've seen used before is:

> **Who owns the intellectual property (IP) in my team’s submission to the hackathon?**
> Unless a team agrees otherwise, the IP in a team’s submission remains the intellectual property of that team. Teams may wish to agree that their submission is Open Source or Creative Commons licensed and is therefore not subject to the proprietary rights of the team members, either individually or collectively. For teams interested in entrepreneurial ventures beyond the hackathon involving products they created during it, we would encourage those teams to have a discussion about the ownership of any IP.

But there are more [legalistic and wordy variants](https://www.goldmansachs.com/careers/blog/posts/multimedia/code-gs-hackathon-terms-and-conditions-2016.pdf) around, too.

We might want to extend this to say something like we'd welcome teams donating open source projects to the AsyncAPI org, or something like that. 

But either way, I think there should be some reassuring statement about ownership.

Background reading on this topic if you're interested:
https://www.wired.com/2013/07/your-friendly-neighborhood-hackathon-might-not-be-so-open-after-all/

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+featured: true
+---
+
+Last year it was the first time we hosted the AsyncAPI Conference. This year it is time to host our first AsyncAPI Hackathon. Many questions are coming from the community, and this article tries to address them.
```
```suggestion
Last year was the first time we hosted the AsyncAPI Conference. This year, we're going to host our first AsyncAPI Hackathon. This article tries to address the many questions coming from our community. 😀
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+1st - 31st of October 2021.
+
+You have an entire month to complete your idea. No usual over-weekend hard work but relaxed month instead.
```
```suggestion
1st - 31st of October, 2021.

We wanted you to have an entire month to complete your idea. This way, you don't have to stress and overwork yourself over the weekend. Instead, enjoy a relaxed month and remember it's about having fun! 🤩 
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+- Ease education and getting started with the project.
+- Bring new AsyncAPI-related solutions that can solve the existing challenges the community is facing with event-driven architectures.
+- Extend existing tools, reuse them in some scenarios, add the extra implementation on top.
```
```suggestion
- Extend existing tools, reuse them for some scenarios, and add the extra implementation on top.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+No.
+
+Anybody can join the Hackathon. Whenever you want. No pressure in case you can't deliver what you planned.
```
```suggestion
No, you do not need to register for the event. 🙂

Anybody can join the Hackathon, whenever they want! No pressure, in case you realize you can't deliver what you had originally planned.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+Anybody can join the Hackathon. Whenever you want. No pressure in case you can't deliver what you planned.
+
+### Is it team competition or for individuals?
```
```suggestion
### Is it a team competition or only for individuals?
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+Both.
+
+You can form a team if you want. Just keep in mind that once you win, the prize also goes to the entire team.
```
```suggestion
You can work solo or form a team if you want! Just keep in mind that if you win with a team, the prize is divided amongst the entire team.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+### I want to participate, but I don't have a good idea
+
+No worries, look at ideas that [others shared](https://github.com/asyncapi/community/discussions/categories/asyncapi-hack-2021-ideas-brainstorming), you can pick one of them or just get some inspiration.
```
```suggestion
No worries, look at the ideas that [community members are sharing in our public community discussion thread](https://github.com/asyncapi/community/discussions/categories/asyncapi-hack-2021-ideas-brainstorming); you can pick one of them and/or get some inspiration. 
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+Yes.
+
+We do not set any limits here. The same idea can be done by multiple teams. Just keep in mind that if there are two excellent submissions for the same idea, I doubt judges will vote for both to let other ideas shine, although this is not an official rule.
```
`"I doubt judges will vote for both to let other ideas shine, although this is not an official rule."`

Sorry, what? this confused me 😕

derberg replied:
I'm just trying to express that if we will have 5 people independently working on https://github.com/asyncapi/community/discussions/55 and event of 2-3 submissions will be super cool, I doubt that judges will choose all of them but most probably just one.

quetzalliwrites replied:
Ok so are you saying that if a single person/team submits 2-3 ideas, judges will prob choose only 1? Because the sentence reads as if you were saying that if many people/teams submitted the same ideas, prob only one team will win. 

So I want to fix this sentence, and make sure it's clearer 😀

derberg replied:
oh, is that what it says 😄 

I'm saying that if multiple people/teams submit the same solution, but of course implemented differently,  then most probably judges will only select one and rest of votes will put on completely different solutions 😄

quetzalliwrites replied:
okidoki, I guess it's ok then... sorry for all the back and forth questions, I was legit trying to make sure I read and understood it all correctly :D

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+We do not set any limits here. The same idea can be done by multiple teams. Just keep in mind that if there are two excellent submissions for the same idea, I doubt judges will vote for both to let other ideas shine, although this is not an official rule.
+
+### What is the scope of the submission? What does it mean I finished my solution on time?
```
```suggestion
### What is the scope of the submission? What does finishing my solution on time really mean?
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+- Screenshots/Video/Diagrams/Code snippets (`anyOf` :wink:) that helps judges to quickly understand the solution and encourage them for further review.
+- Link to the repository where the full solution can be found.
+
+This is not it. If you win, you need to present your idea at [AsyncAPI Conference in November](http://conference.asyncapi.com/)
```
```suggestion
If you win, you get to present your idea at our upcoming 2021 [AsyncAPI Conference in November](http://conference.asyncapi.com/)
```

I think it accidentally sounds a little short/rude to start with "this is not it." That's why I removed it. Also overall was trying to make things sound friendlier.

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+A selected group that will represent the community will review all solutions. They will be obligated to only provide YES comments for ideas they liked the most and explain why. Each judge has 3 votes. 
+
+Submissions and scores are public, and we need to reserve a right for judges to not make a public NO comment for the respect of all involved parties.
```
```suggestion
Submissions and scores are public, and we reserve the right for judges to not make public NO comments, for the respect of all involved parties.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+Submissions and scores are public, and we need to reserve a right for judges to not make a public NO comment for the respect of all involved parties.
+
+Solution with the highest score wins.
```
```suggestion
The solution with the highest score wins! 😀
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+### How do I get help? Who can help with further understanding of AsyncAPI and the entire tooling landscape?
+
+- Join one of `Contributor First` meetings. These will be streamed throughout the entire October. Add this calendar with [Google Calendar format](https://calendar.google.com/calendar/embed?src=tbrbfq4de5bcngt8okvev4lstk%40group.calendar.google.com) or [iCal format](https://calendar.google.com/calendar/ical/tbrbfq4de5bcngt8okvev4lstk%40group.calendar.google.com/public/basic.ics). `Contributor Firs ...
```
```suggestion
- Join one of `Contributor First` meetings. These will be streamed throughout the entire month of October, 2021. Add this calendar with [Google Calendar format](https://calendar.google.com/calendar/embed?src=tbrbfq4de5bcngt8okvev4lstk%40group.calendar.google.com) or [iCal format](https://calendar.google.com/calendar/ical/tbrbfq4de5bcngt8okvev4lstk%40group.calendar.google.com/public/basic.ics).  The `Contributor First` meetings will take place every Wednesday, twice a day, at 8AM UTC and 4PM UTC.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+### What prizes are there?
+
+$5000 for the winner, $3000 for runner-up, and $1000 for 3rd.
```
```suggestion
$5000 for the winner, $3000 for the runner-up, and $1000 for 3rd place.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,98 @@
+
+There will also be additional swag-packs for other participants.
+
+### How do I get the money or swag?
```
```suggestion
### How do I get the winning prize money or swag?
```

quetzalliwrites commented:
> @alequetzalli wanna have a look?
> 
> @arjungarg07 & @KhudaDad414 you had some questions, some of them I've put in FAQ. Have a look? especially if there are some answers that are not addressed

I HAD A LOOK ! 👀 😀🦄

derberg committed:
Apply suggestions from code review from Alejandra

Co-authored-by: Alejandra Quetzalli  <19964402+alequetzalli@users.noreply.github.com>

derberg committed:
Merge branch 'master' into hackathon-faq

derberg commented:
@alequetzalli Thanks so much!!!

derberg committed:
info about IP and license

derberg commented:
@dalelane super valid point, thanks! I added a separate section, tried to make a sound as simple as possible + also added that it is required for the project to have a license

@alequetzalli wanna have another look on ☝🏼 additions?

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,102 @@
+### What is the scope of the submission? What does finishing my solution on time really mean?
+
+- Your solution must be fully open-sourced and stored in a repository under your GitHub profile. Please do not use Bitbucket or GitLab as it will increase complexity for our community members that will review submissions.
+- Your repository must have [an open source](https://choosealicense.com/) license. If that helps, at AsyncAPI Initiative we always use Apache 2.0 :wink:.
```
```suggestion
- Your repository must have [an open source](https://choosealicense.com/) license. If it helps, at AsyncAPI Initiative we always use Apache 2.0 :wink:.
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,102 @@
+- Screenshots/Video/Diagrams/Code snippets (`anyOf` :wink:) that helps judges to quickly understand the solution and encourage them for further review.
+- Link to the repository where the full solution can be found.
+
+If you win, you get to present your idea at our upcoming 2021 [AsyncAPI Conference in November](http://conference.asyncapi.com/)
```
```suggestion
If you win, you get to present your idea at our upcoming 2021 [AsyncAPI Conference in November](http://conference.asyncapi.com/).
```

quetzalliwrites commented on pages/blog/hackathon-faq.md:
```diff
@@ -0,0 +1,102 @@
+
+For swags, we will ask you directly to share your postal address with us that we will share with the agency responsible for swag-pack production.
+
+### Who owns the intellectual property (IP) 
```
```suggestion
### Who owns the intellectual property (IP)?
```

derberg committed:
Apply suggestions from code review

Co-authored-by: Alejandra Quetzalli  <19964402+alequetzalli@users.noreply.github.com>

derberg committed:
Merge branch 'master' into hackathon-faq

derberg committed:
Merge branch 'master' into hackathon-faq
